### DISCUSSION TITLE
20241025 DAB still not working (grayed out)

### DISCUSSION THREAD

**Discussion by hjfisch** [rern/rAudio#1794] [Bug Reports]
"Found 1 device(s):
  0:  Nooelec, SMArt XTR v5, SN: 00000001

Using device 0: Generic RTL2832U OEM
Detached kernel driver
Found Elonics E4000 tuner
Supported gain values (14): -1.0 1.5 4.0 6.5 9.0 11.5 14.0 16.5 19.0 21.5 24.0 29.0 34.0 42.0 
Sampling at 2048000 S/s.
Benchmarking E4000 PLL...
Signal caught, exiting!
[E4K] PLL not locked for 51000000 Hz!
[E4K] PLL not locked for 2173000000 Hz!
[E4K] PLL not locked for 1087000000 Hz!
[E4K] PLL not locked for 1259000000 Hz!
E4K range: 52 to 2172 MHz
E4K L-band gap: 1087 to 1259 MHz
Reattached kernel driver

# systemctl status mediamtx
* mediamtx.service - MediaMTX
     Loaded: loaded (/usr/lib/systemd/system/mediamtx.service; enabled; preset: disabled)
     Active: active (running) since Fri 2024-10-25 19:57:02 CEST; 29s ago
 Invocation: 13d52278e9db4d65915502c638f50ae0
   Main PID: 8566 (mediamtx)
      Tasks: 8 (limit: 4465)
        CPU: 74ms
     CGroup: /system.slice/mediamtx.service
             └─8566 /usr/bin/mediamtx /etc/mediamtx/mediamtx.yml

Oct 25 19:57:02 hjfSound systemd[1]: Started MediaMTX.
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF MediaMTX v1.9.3
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF configuration loaded from /etc/mediamtx/mediamtx.yml
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF [RTMP] listener opened on :1935
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF [HLS] listener opened on :8888
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF [WebRTC] listener opened on :8889 (HTTP), :8189 (ICE/UDP)
Oct 25 19:57:02 hjfSound mediamtx[8566]: 2024/10/25 19:57:02 INF [SRT] listener opened on :8890 (UDP)"

DAB-Stick is found now and can be enabled but DAB Radio is grayed out and no scan is possible .
No dabradio folder in /srv/http/data . I made this folder , but nothing has changed .

**Comment by rern** [comment]
Please post output of:
`rtl_test -t`
`dab-scanner-rtlsdr`

**Comment by hjfisch** [comment]
hjfSound:~  #  rtl_test -t
Found 1 device(s):
  0:  Nooelec, SMArt XTR v5, SN: 00000001

Using device 0: Generic RTL2832U OEM
Found Elonics E4000 tuner
Supported gain values (14): -1.0 1.5 4.0 6.5 9.0 11.5 14.0 16.5 19.0 21.5 24.0 29.0 34.0 42.0
Sampling at 2048000 S/s.
Benchmarking E4000 PLL...
[E4K] PLL not locked for 51000000 Hz!
[E4K] PLL not locked for 2176000000 Hz!
[E4K] PLL not locked for 1089000000 Hz!
[E4K] PLL not locked for 1261000000 Hz!
E4K range: 52 to 2175 MHz
E4K L-band gap: 1089 to 1261 MHz

dab-scanner-rtlsdr
-bash: hjfSound:~: command not found
-bash: syntax error near unexpected token `('
-bash: 0:: command not found
-bash: Using: command not found
-bash: Found: command not found
-bash: syntax error near unexpected token `('
-bash: Sampling: command not found
-bash: Benchmarking: command not found
-bash: [E4K]: command not found
-bash: [E4K]: command not found
-bash: [E4K]: command not found
-bash: [E4K]: command not found
-bash: E4K: command not found
-bash: E4K: command not found
-bash: hjfSound:~: command not found
dab_scanner V 1.0alfa,
Copyright 2018 J van Katwijk, Lazy Chair Computing
2018 Hayati Ayguen
2019 J van Katwijk
2020 J van Katwijk
                        dab-scanner options are
                        -F filename      in case the output is to a file
                        -j               output data in json format
                        -d number        amount of time to look for time synchr
                        -D number        amount of time to look for full sync
                        -M Mode          Mode is 1, 2 or 4. Default is Mode 1
                        -B Band          Band is either L_BAND or BAND_III (default)
                        -C start channel the start channel, default: 5A
                        -R filename     raw dump of the input data
        for hackrf:
                          -v vgaGain
                          -l lnaGain
                          -p number     ppm offset
        for SDRplay:
                          -G Gain reduction in dB (range 20 .. 59)
                          -L lnaState (depends on model chosen)
                          -Q autogain (default off)
                          -p number      ppm offset
        for rtlsdr:
                          -G number             gain, range 0 .. 100
                          -Q autogain (default off)
                          -p number     ppm offset
        for airspy:
                          -G number             gain, range 1 .. 21
                          -b set rf bias
                          -p number      ppm Correction
        for limesdr:
                          -G number      gain
                          -X antenna selection
                          -C channel
